README.md

```diff
@@ -1,23 +1,118 @@
 # clp-bench
-clp-bench is a tool for benchmarking [CLP] as well as other log management tools. The tool itself is
-a Python package, and we also provide a [web interface][ui] for viewing results.
 
-The methodology for the benchmarks is described [here](docs/methodology.md).
+**clp-bench** is a benchmarking tool designed for [CLP] and other log management systems. It
+functions as a Python package and includes a [web interface][ui] for displaying benchmark results.
+
+For a detailed description of the benchmarking methodology, see
+[this document](docs/methodology.md).
 
 ## Requirements
 
-* Docker
-* Python v3.10 or higher
+- Docker
+- Python v3.10 or higher
 
-# Set up
+# Setup
 
 ```shell
 python3 -m venv venv
 . venv/bin/activate
 pip install -e .
 ```
 
-You can use `clp-bench --help` to see usage instructions.
+To view usage instructions, run `clp-bench --help`.
```

Comment:
```suggestion
To view usage instructions, run `clp-bench --help`.

# Results

You can view the current benchmark results [here][]. The benchmark currently evaluates
ingestion and query performance for the following tools:

Don't see a tool here? Feel free to file a [GitHub issue][] for one or follow this
[guide][] for how to add one.

For a detailed description of the benchmarking methodology, see [here](docs/methodology.md).
```

In the suggestion above, you should also:
* Add the link to the benchmarks page.
* Add a list of the benchmarked tools and their versions.
* Add the link to the issue creation form.
* Move the contributing section below into docs (perhaps call the page `Adding tools` and the file `adding-tools.md`).
* Add a link to the new page above.

Reply:
Yep